broker/broker-routemanager/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<blueprint xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.2.0"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0 https://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
+
+    <camelContext xmlns="http://camel.apache.org/schema/blueprint" id="routedeployer-context"/>
+
+    <ext:property-placeholder placeholder-prefix="$[" placeholder-suffix="]"/>
+
+    <reference id="jmsConnectionFactory" interface="javax.jms.ConnectionFactory"/>
+
+    <bean id="jmsConsumerConnectionResource"
+          class="org.apache.camel.component.sjms.jms.ConnectionFactoryResource"
+          destroy-method="drainPool">
+        <argument value="10"/>
+        <argument ref="jmsConnectionFactory"/>
+    </bean>
+
+    <bean id="sjms" class="org.apache.camel.component.sjms.SjmsComponent">
```

Comment:
Yes, unless we add more components to the camel context. I think that could be done dynamically, as long as we can reference this camel context. Maybe we'll need to change this blueprint to export the camel context or make an easy way to add components to the context inside this bundle (some new bean watches for `CamelComponents` that are dropped into `deploy` and adds them to this camel context?)